port/common/omrport.c

```diff
@@ -259,6 +259,31 @@ static OMRPortLibrary MainPortLibraryTable = {
 	omrmmap_protect, /* mmap_protect */
 	omrmmap_get_region_granularity, /* mmap_get_region_granularity */
 	omrmmap_dont_need, /* mmap_dont_need */
+#if !defined(OMR_OS_WINDOWS)
+	omrshsem_params_init, /* shsem_paramemeters_init */
+	omrshsem_startup, /* shsem_startup */
+	omrshsem_shutdown, /* shsem_shutdown */
+	omrshsem_open, /* shsem_open */
+	omrshsem_post, /* shsem_post */
+	omrshsem_wait, /* shsem_wait */
+	omrshsem_getVal, /* shsem_getVal */
+	omrshsem_setVal, /* shsem_setVal */
+	omrshsem_close, /* shsem_close */
+	omrshsem_destroy, /* shsem_destroy */
+	omrshsem_deprecated_startup, /* shsem_deprecated_startup */
```

Comment:
Do we need to move the deprecated functions?

Reply:
Yes, these functions are necessary as they will be consumed by other shared cache files which are included in other PR's.

Reply:
Thanks for the explanation!